cores/arduino/main.cpp

```diff
@@ -45,6 +45,9 @@ int main( void )
 
 	for (;;)
 	{
+		// Define by Ethernet library. It is defined as __weak.
+		stm32_eth_scheduler();
```

Comment:
The same would happen if you put this call in the loop, so this does not answer my question.
Also this seems here that it will be called for each loop for a board that supports it, even if I don't use Ethernet , is that right ?